src/app/pages/MediaArticlePage/SecondaryColumn.jsx

```diff
@@ -45,12 +45,12 @@ const SecondaryColumn = ({ pageData }) => {
         <ResponsiveComponentWrapper data-testid="features">
           <FeaturesAnalysis
             content={featuresContent}
-            parentColumns={gridColumnsSecondary}
+            parentColumns={{}}
```

Comment:
Passing an empty object here so that the Grid component isn't used further down in the `CpsOnwardJourney` component: https://github.com/bbc/simorgh/blob/17b68f40938a4e45d256d8b5bde3e28898c50c87/src/app/legacy/containers/CpsOnwardJourney/index.jsx#L187-L196

Reply:
Not an issue really as the secondary column will soon have a different set of data coming from TIPO, so will be using the Promo compound components, rather than the CpsOnwardJourney ones here.